materialize-bigquery/.snapshots/TestSpecification

```diff
@@ -11,8 +11,8 @@
       },
       "credentials_json": {
         "type": "string",
-        "title": "Credentials",
-        "description": "Google Cloud Service Account JSON credentials in base64 format.",
+        "title": "Service Account JSON",
+        "description": "The JSON key of the service account to use for authorization.",
```

Comment:
nit: s/key/credentials/ perhaps? The whole JSON credentials blob is more than just the private key.

Reply:
Seems like it would be better, thanks!